app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,64 @@
+<style>
+.buttons {
+  display: flex;
+}
+.margin {
+  margin-right: 8px;
+}
+</style>
+
+<%= render partial: 'application/header', layout: 'header' %>
+<%= render partial: 'application/body', layout: 'body' %>
+
+<div class="myDiv" >
+<h1>Name on Application: <%= @application.name %></h1>
+</div>
+<div class="body" >
+<p><u>Address</u>: <%= @application.full_address %> </p>
+<p><u>Why would you make a good home?</u>: <%= @application.description %> </p>
+<p><u>Status</u>: <%= @application.status %> </p>
+
+<p><u>Pets I'd Like to Adopt</u>:</p>
+<% @application.application_pets.each do |pet| %>
+<div class= "buttons">
+  <% if pet.application_pet_status == "Pending" && pet.adoptable == true %>
+    <%= form_with url:"/application_pets/#{@application.id}", method: :patch, local: :true do |f| %>
```

Comment:
See note in controller about this path -- it's expected that the ID you pass here is the application_pet record ID so this path might've been a better choice (but both are RESTful): "applications/:app_id/pets/:pet_id"